models/events/TestTakerRemovedEvent.php

```diff
@@ -22,11 +22,36 @@
 
 namespace oat\taoTestTaker\models\events;
 
+use oat\tao\model\webhooks\WebhookSerializableEventInterface;
+
 /**
  * Class TestTakerRemovedEvent
  * @package oat\taoTestTaker\models\events
  */
-class TestTakerRemovedEvent extends AbstractTestTakerEvent
+class TestTakerRemovedEvent extends AbstractTestTakerEvent implements WebhookSerializableEventInterface
 {
     const EVENT_NAME = __CLASS__;
+
+    /**
+     * @inheritDoc
+     */
+    public function getWebhookEventName()
+    {
+        return 'test-taker-removed';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function serializeForWebhook()
+    {
+        return [
+            'name' => $this->getWebhookEventName(),
+            'payload' => [
+                'testTakerUri' => $this->testTakerUri,
+                'unit' => 1,
+                'tenant' => defined('LOCAL_NAMESPACE') ? LOCAL_NAMESPACE : ROOT_URL,
```

Comment:
As we have multiple event that implies tenant notion, I would suggest to centralize to a dedicated service

Reply:
Yes, true. This is just for testing now. As soon we have an answer from @llecaque we can move it to better location. :)

Reply:
@siwane What do you think about tao-core?

Reply:
It was made a configuration